content/en/blog/_posts/2024-09-30-gateway-api-v1.2/index.md

```diff
@@ -0,0 +1,418 @@
+          port: 80
+        percent: 42
+```
+
+There's also a `fraction` stanza which can be used in place of `percent`, to allow for more fine-grained control over the mirroring percentage:
+
+```yaml
+    ...
+    filters:
+    - type: RequestMirror
+      requestMirror:
+        backendRef:
+          name: color-mirror
+          port: 80
+        fraction:
+          numerator: 1
+          denominator: 10000
+```
+
+This configuration mirrors 1 in 10,000 requests to the `color-mirror` backend,
+which may be relevant with very high request rates.
+
+#### [BackendTLSPolicy Client Certificates](https://gateway-api.sigs.k8s.io/geps/gep-3155)
+
+Last but not least for new experimental features, Gateway API 1.2.0 introduces
+the ability to configure client certificates for backend connections, enabling
+mutual TLS authentication. The following BackendTLSPolicy configuration
+specifies a client certificate to use when connecting to the `color` backend:
+
+```yaml
+...needs work here...
+```
+
+### More Changes
+
+For a full list of the changes included in this release, please refer to the
+[v1.2.0 release notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.2.0).
+
+## Project Updates
+
+Beyond the technical, the 1.2 release also marks a few milestones in the life
+of the Gateway API project itself.
+
+### Release Process Improvements
+
+Gateway API has never been intended to be a static API, and as more projects
+use it as a component to build on, it's become clear that we need to bring
+some more predictability to Gateway API releases. To that end, we're pleased -
+and a little nervous! - to announce that we've formalized a new release
+process:
+
+- **Scoping** (4-6 weeks): maintainers and community determine the set of
+  features we want to include in the release. A particular emphasis here is
+  getting features _out_ of the Experimental channel -- ideally this involves
+  moving them to Standard, but it can also mean removing them.
+
+- **GEP Iteration and Review** (5-7 weeks): contributors write or update
+  Gateway Enhancement Proposals (GEPs) for features accepted into the release,
+  with emphasis on getting consensus around the design and graduation criteria
+  of the feature.
+
+- **API Refinement and Documentation** (3-5 weeks): contributors implement the
+  features in the Gateway API controllers and write the necessary
+  documentation.
+
+- **SIG-Network Review and Release Candidates** (2-4 weeks): maintainers get
+  the required upstream review, build release candidates, and release the new
+  version.
+
+Gateway API 1.2.0 is the first release to use the new process, and although
+there are the usual rough edges of anything new, we believe that it went well.
+To that end, we've just started the Scoping phase for Gateway API 1.3, which
+will end on November 19, 2024, with the release expected around the end of
+January 2025.
+
+### `gwctl` Moves Out
+
+The `gwctl` CLI tool has moved into its very own repository,
```

Comment:
@gauravkghildiyal did you want to add more here?

Reply:
I'm going to resolve this, but @gauravkghildiyal, hit me up on Slack if you want to add something!